build-tools/automation/yaml-templates/build-macos.yaml

```diff
@@ -80,11 +80,3 @@ stages:
         xaSourcePath: $(System.DefaultWorkingDirectory)/xamarin-android
         artifactName: ${{ parameters.buildResultArtifactName }}
         includeBuildResults: true
-
-    - template: run-xaprepare.yaml
-      parameters:
-        arguments: --s=DetermineApplicableTests
```

Comment:
So we could probably delete `--s=DetermineApplicableTests` in `xaprepare`, but maybe we don't have to do that here.